crates/subspace-farmer/src/legacy_multi_plots_farm.rs

```diff
@@ -28,11 +28,11 @@ pub struct Options<C> {
     pub object_mappings: ObjectMappings,
     pub reward_address: PublicKey,
     pub bootstrap_nodes: Vec<Multiaddr>,
-    pub listen_on: Vec<Multiaddr>,
     /// Enable DSN subscription for archiving segments.
     pub enable_dsn_archiving: bool,
     pub enable_dsn_sync: bool,
     pub enable_farming: bool,
+    pub relay_server_node: Arc<Node>,
```

Comment:
Node is clonable already and has `Arc` internally, no need to wrap it in another `Arc` on top of that.